ui/gtk/main_window.py

```diff
@@ -0,0 +1,73 @@
+import gi
+
+gi.require_version('Gtk', '3.0')
+
+from gi.repository import Gdk, GObject, Gtk, GdkPixbuf
+
+
+class Handler:
+    @staticmethod
+    def on_destroy(*args):
+        Gtk.main_quit()
+
+
+class Components:
+    def __init__(self, builder):
+        self._allComponents = {}
```

Comment:
Python conventions dictate that variable names must follow snake_case 
```suggestion
        self._all_components = {}
```